src/courseware/course/sequence/sequence-navigation/UnitButton.jsx

```diff
@@ -24,6 +24,23 @@ const UnitButton = ({
     onClick(unitId);
   }, [onClick, unitId]);
 
+  const handleKeyDown = (event) => {
+    if (event.key === 'Enter' || event.key === ' ') {
+      onClick(unitId);
+
+      const performFocus = () => {
+        const targetElement = document.getElementById('bookmark-button');
+        if (targetElement) {
+          targetElement.focus();
+        }
+      };
```

Comment:
[question]: We currently use a similar implementation three times in the project code
Does it make sense to write a custom generic react hook for this functionality?

Reply:
two times I checked.
not sure, at your discretion (function so little)